src/pages/journal/journal.html

```diff
@@ -5,4 +5,10 @@
 <ion-content padding>
   <button ion-button navPush="ContactDetailsPage">Contact Details</button>
   <button ion-button navPush="WaitingRoomPage">Waiting Room</button>
+
+ <ion-list>
+  <ion-card *ngFor="let s of journalSlot">
```

Comment:
Probably worth a test to make sure the `ngFor` has generated a card per entry in your test dataset.

Reply:
Happy to merge once we get this test migrated to Jasmine + Karma as per DVSA guidelines.

Reply:
Happy to approve with this test as Jest for now, as the testing framework discussion is still ongoing.